source/elens.f90

```diff
@@ -383,35 +381,41 @@ end subroutine elens_postInput
 !  - total beam energy [MeV]
 !  - outer radius [mm]
 ! ================================================================================================ !
-real(kind=fPrec) function eLensTheta( len, Int, Ekin, Etot, r2 )
+subroutine eLensTheta( j, Etot )
 
   use floatPrecision
   use mathlib_bouncer
   use numerical_constants
   use physical_constants
   use crcoall
   use mod_common
+  use mod_hions, only : nucm0, zz0
 
   implicit none
 
-  real(kind=fPrec) gamma, beta_e, beta_b, brho, len, Int, Ekin, Etot, r2
+  integer j
+  real(kind=fPrec) gamma, beta_e, beta_b, brho, Etot
 
-  gamma  = Ekin*c1m3/pmae+1 ! from kinetic energy
+  gamma  = elens_Ek(j)*c1m3/pmae+one ! from kinetic energy
```

Comment:
This line needs brackets for consistent rounding.

Reply:
done - I will anyway review the entire code of elenses to make sure that there are no other left-overs of this type